### PR TITLE
lesson 9 classes

### PR DESCRIPTION
**Pull request by mariusamsarius**
add my homework for lesson 9

### CODE REVIEW

File: lesson_9/lesson9_classes.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def __init__(self, category, brand, model, stock, price):
+        self.category = category
+        self.brand = brand
+        self.model = model
+        self.stock = stock
+        self.price = price
+
+    def category_check(self):
+        if self.category is 'Laptop':
+            return f'Category: Computers'
```
Comment (lorena91) [inline lesson_9/lesson9_classes.py]: You could also handle the else case

File: lesson_9/lesson9_classes.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    def special_feature(self):
+        return self.feature
+
+
+lenovo = Laptop('Laptop', 'Lenovo', 'IdeaPad L340', 5, '6999 RON', '17,3"', 'Touchscreen')
+hp = Laptop('Laptop', 'HP', 'Pavilion', 2, '4999 RON', '15,6"', '4K')
+asus = Laptop('Laptop', 'Asus', '8565U', 4, '2999 RON','14"', 'Ultraportable')
+dell = Laptop('Laptop', 'Dell', 'Alienware', 0, '9999 RON','17"', 'Gaming')
+
```
Comment (lorena91) [inline lesson_9/lesson9_classes.py]: I think there should be 5 instances

**Review by lorena91 — COMMENTED** [review]
Looking good and clean! :+1:

**Comment by Liitle** [comment]
Hi, Please remove the files from lesson_7 :smiley:. Also, you can create a file in the root of the project named `.gitignore` and add in that file `.idea` this operation will exclude PyCharm config files `.idea`.
If you want you can also exclude `.idea` files manually. 

If you are having trouble making the changes let me know and I will help

File: lesson_9/dragla_marius/lesson9_classes.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            return 'Category: Others'
+
+    def on_stock(self):
+        if self.stock > 0:
+            return 'Availability: On stock'
+        else:
+            return 'Availability: Out of stock'
+
+    def price(self):
+        return self.price
```
Comment (Liitle) [inline lesson_9/dragla_marius/lesson9_classes.py]: Usually, the methods need to contain some form of functionality. If you just return the value the function call is similar to just reading the attribute from the instance.

Laptop.price will be the same as self.price. Also having the same name for methods and attributes is not good practice. Try to add some functionality to the method and change the name :smile:

File: lesson_9/dragla_marius/lesson9_classes.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    def __init__(self, category, brand, model, stock, price, display, feature):
+        self.display = display
+        self.feature = feature
+        super().__init__(category, brand, model, stock, price)
+
+    def description(self):
+        return f'{self.category} {self.brand} {self.model} {self.display}'
+
+    def special_feature(self):
```
Comment (Liitle) [inline lesson_9/dragla_marius/lesson9_classes.py:44]: same as the comment above

**Review by Liitle — COMMENTED** [review]
Nice work, clean code and to the point. Some small changes are required, see the comments bellow

File: lesson_9/lesson9_classes.py (hunk 2 above)
Comment (mariusamsarius) [reply lesson_9/lesson9_classes.py]: added one more instance

File: lesson_9/dragla_marius/lesson9_classes.py (hunk 3 above)
Comment (mariusamsarius) [reply lesson_9/dragla_marius/lesson9_classes.py]: added functionality to convert price to euro

File: lesson_9/dragla_marius/lesson9_classes.py (hunk 4 above)
Comment (mariusamsarius) [reply lesson_9/dragla_marius/lesson9_classes.py:44]: done

**Comment by mariusamsarius** [comment]
> Hi, Please remove the files from lesson_7 smiley. Also, you can create a file in the root of the project named `.gitignore` and add in that file `.idea` this operation will exclude PyCharm config files `.idea`.
> If you want you can also exclude `.idea` files manually.
> 
> If you are having trouble making the changes let me know and I will help

@Liitle - I have deleted lesson 7 and added the .idea line in gitignore, but I am not sure how to delete the .idea folder on my branch. If I do a pull from git to my local project, I cannot see the .idea folder anywhere. Can you please advise how I can remove it from the git branch?

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**